core/metaobjectrepository.h

```diff
@@ -148,6 +148,12 @@ class GAMMARAY_CORE_EXPORT MetaObjectRepository
                         #Getter, \
                         &Class::Getter));
 
+/** Register a read-only property (non const getter) for class @p Class. */
+#define MO_ADD_PROPERTY_RO_NC(Class, Type, Getter) \
```

Comment:
This entire thing should not be needed when looking at the version conditions below.